config/initializers/filter_parameter_logging.rb

```diff
@@ -51,5 +51,8 @@
 # Assessment filters
 Rails.application.config.filter_parameters += %i[symptomatic]
 
-# History Items
+# History filters
 Rails.application.config.filter_parameters += %i[comment]
+
+# Vaccine filters
+Rails.application.config.filter_parameters += %i[group_name product_name administration_date dose_number notes ]
```

Comment:
Looks like there is an unnecessary space between `notes` and the ending bracket.

Reply:
Fixed here: 360c412